src/main/java/main/logic/command/CommandFactory.java

```diff
@@ -0,0 +1,54 @@
+package main.logic.command;
+
+
+import main.logic.handler.*;
```

Comment:
Try to import the direct class/classes rather than using star import

Reply:
Right, missed this, thanks!